### ISSUE TITLE
local aliases to cabals

### ISSUE THREAD

**Issue by noffle** [cabal-club/cabal#70]
It'd be neat if I ran

```
cabal --key cabal://abcdef123456 --alias friends
```
and then I could subsequently run

```
cabal friends
```
and have that cabal open.

**Comment by kodedninja** [comment]
I wonder if this should be part of [cabal](https://github.com/cabal-club/cabal) or [cabal-node](https://github.com/cabal-club/cabal-node) within an API for local storage. Opinions?

**Comment by todrobbins** [comment]
@kodedninja the design would need to be figured out, but giving cabals on Cabal Desktop aliases would be great and probably fit in [cabal-core].

**Comment by noffle** [comment]
I think it's a good feature for cabal-client. My thinking is that Core is the place for managing the p2p database, and Client could be where all local preferences and local state are stored.

**Comment by cblgh** [comment]
+1

**Comment by todrobbins** [comment]
@noffle makes a better point 👍

**Comment by cblgh** [comment]
can be implemented in #89's config file as another key `aliases` that maps a cabal id to an alias. 

when starting cabal we will check if there is a config file (locally, or in .cabal). if there is, we will see if the key we were given was in `aliases`. if it was, we'll join the corresponding `cabal://` key. if it wasn't present, we'll assume that what was passed in was a cabal key
